### ISSUE TITLE
New/Edit/Delete buttons no longer customized to model name

### ISSUE THREAD

**Issue by fredngo** [activeadmin/activeadmin#4954]
Hello,

I recently upgraded from activeadmin 1.0.0.pre4 to 1.0.0 and noticed that the New/Edit/Delete buttons no longer have the proper text.

In 1.0.0.pre4 it used to say "New Company":

<img width="1020" alt="aa_before" src="https://cloud.githubusercontent.com/assets/11791/25765817/f563f318-31bc-11e7-8b73-20d6f0f48474.png">

In 1.0.0 (and 1.0.0.pre5), it now says "New Model".

<img width="1035" alt="aa_after" src="https://cloud.githubusercontent.com/assets/11791/25765838/0bcaacdc-31bd-11e7-95a6-6f64771e1ffc.png">

It appears to be a minor issue, but non-technical admins are sure to make a fuss about this.

Thanks for any insights.

**Comment by Fivell** [comment]
@fredngo please share some code

**Comment by tonyvince** [comment]
We could not reproduce the issue with v1.0.0 and the New button works as expected. Could you please share more details about how you upgraded etc..

**Comment by fredngo** [comment]
Hi @tonyvince, there isn't much to share about how I upgraded, I simply changed active admin from version `1.0.0.pre4` to `1.0` in my Gemfile and then `bundle update activeadmin`.

@Fivell: I can share my Companies admin code below:

```ruby
ActiveAdmin.register Company, as: I18n.t('active_admin.names.company') do

  menu label: I18n.t('active_admin.labels.companies')

  filter :name

  index do
    column :name
    actions
  end

end
```

Even if I remove the I18n and use simply the following, it still says "New Model" in the button:

```ruby
ActiveAdmin.register Company do

  filter :name

  index do
    column :name
    actions
  end

end
```

**Comment by Fivell** [comment]
@fredngo,  I can't reproduce. Try to create failing spec or bug report template 
https://github.com/activeadmin/activeadmin/blob/master/CONTRIBUTING.md#4-did-you-find-a-bug

**Comment by fredngo** [comment]
@Fivell : I can take a look at the code here and try to fix it in a pull request, if you can point me towards the file/method that is responsible for rendering the button?

**Comment by Fivell** [comment]
@fredngo , take a look https://github.com/activeadmin/activeadmin/blob/11c1253d2d228cdae27537fff16d75eb091dca69/lib/active_admin/resource/action_items.rb#L65

**Comment by fredngo** [comment]
It turns out that en-CA is my default locale but there wasn't a corresponding en-CA translation.

I added an en-CA translation file and also updated the en-GB file.

https://github.com/activeadmin/activeadmin/pull/4972

**Comment by Fivell** [comment]
closed by #4972
